deploy/crds/pulpproject_v1alpha1_pulp_cr.travis.yaml

```diff
@@ -3,6 +3,11 @@ kind: Pulp
 metadata:
   name: example-pulp
 spec:
+  project: pulp
+  # The image name (repo name) for the pulp image.
+  image: pulp
+  # The image tag for the pulp image.
```

Comment:
The comment here describes "what", not "why". The "what" is already documented in .default.yaml, so we should remove the comment.